.github/ISSUE_TEMPLATE/3_ci_error_template.yml

```diff
@@ -18,10 +18,10 @@ body:
     id: ci-test-type
     attributes:
       label: CI Test Type
-      description: In which CI test did this failure occur?
+      description: >
+        In which CI test did this failure occur? For E2E or Acceptance test failures,
+        please use our dedicated template.
```

Comment:
Maybe add "instead".

Also, could you link them to that template? Otherwise they might not know what this is referring to.

Reply:
how do you link to issue types in the issue submission UI? I didn't realize that was possible. I can link to the template yml file, but that's not too useful since you can't submit issues there

Reply:
Oh sorry, yes, I'm not sure you can, my bad.

Instead, could you give them more specific instructions then? E.g. click X and select the option "AAA"?